doc/orchestrator.rst

```diff
@@ -1,207 +1,724 @@
+  # Use the SmartRedis client to place tensor in the standalone Orchestrator
+  driver_client.put_tensor("tensor_1", local_array)
+
+Model Initialization
+--------------------
+In the next stage of the experiment, we configure and create
+a SmartSim ``Model`` and specify the application script path during ``Model`` creation.
+
+Step 1: Configure
+'''''''''''''''''
+In this example experiment, the ``Model`` application is a Python script as defined in section:
+:ref:`Application Script<standalone_orch_app_script>`. Before initializing the ``Model`` object, we must use
+``Experiment.create_run_settings()`` to create a ``RunSettings`` object that defines how to execute
+the ``Model``. To launch the Python script in this example workflow, we specify the path to the application
+file `application_script.py` as the `exe_args` parameter and the executable `exe_ex` (the Python
+executable on this system) as `exe` parameter. The ``Experiment.create_run_settings()`` function
+will return a ``RunSettings`` object that can then be used to initialize the ``Model`` object.
+
+.. note::
+  Change the `exe_args` argument to the path of the application script
+  on your file system to run the example.
+
+Use the ``RunSettings`` helper functions to
+configure the the distribution of computational tasks (``RunSettings.set_nodes()``). In this
+example, we specify to SmartSim that we intend the ``Model`` to run on a single compute node.
+
+.. code-block:: python
+
+  # Initialize a RunSettings object
+  model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/path/to/application_script.py")
+  model_settings.set_nodes(1)
+
+Step 2: Initialize
+''''''''''''''''''
+Next, create a ``Model`` instance using the ``Experiment.create_model()`` factory method.
+Pass the ``model_settings`` object as an argument to the ``create_model()`` function and
+assign the returned ``Model`` instance to the variable `model`:
+
+.. code-block:: python
+
+  # Initialize the Model
+  model = exp.create_model("model", model_settings)
+
+File Generation
+---------------
+To created an isolated output directory for the ``Orchestrator`` and ``Model``, invoke ``Experiment.generate()`` via the
+``Experiment`` instance `exp` with `standalone_orchestrator` and `model` as input parameters:
+
+.. code-block:: python
+
+  # Create the output directory
+  exp.generate(standalone_orchestrator, model)
+
+.. note::
+  Invoking ``Experiment.generate(standalone_orchestrator, model)`` will create two directories:
+  `standalone_orchestrator/` and `model/`. Each of these directories will store
+  two output files: a `.out` file and a `.err` file.
+
+.. note::
+  It is important to invoke ``Experiment.generate()`` with all ``Experiment`` entity instances as input
+  as well as before any entities have been launched. This will ensure that the output files are
+  organized in the main ``experiment-name/`` folder. In this example, the ``Experiment`` folder is named
+  `getting-started/`.
```

Comment:
I might suggest that I make it shorter, just so I can still walk through every step of the Experiment, and still link to the Model doc - it might be confusing to a user if I fully skip the model init section since Im providin a handl holdy example